c/pthread-C-DAC/pthread-producer-consumer.c

```diff
@@ -0,0 +1,227 @@
+/*****************************************************************************
+                        C-DAC Tech Workshop : hyPACK-2013                             
+ October 15-18,2013
+ 
+  Example     : pthread-producer-consumer.c
+  
+  Objective   : To illustrate producer-consumer problem using pthreads.
+ 
+  Input       : Nothing.
+ 
+  Output      : Sequence of Produced and Consumed items.
+                                                                                                                         
+  Created     : MAY-2013 
+  E-mail      : hpcfte@cdac.in          
+                                
+****************************************************************************/
+ 
+#include <pthread.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <stdlib.h>
```

Comment:
After fixing all the other parts:
Could you also add a second file containing the **preprocessed source**?